airflow/bin/cli.py

```diff
@@ -888,7 +882,7 @@ def webserver(args):
             '-b', args.hostname + ':' + str(args.port),
             '-n', 'airflow-webserver',
             '-p', str(pid),
-            '-c', 'python:airflow.www.gunicorn_config',
+            # '-c', 'python:airflow.www.gunicorn_config',
```

Comment:
Why is this commented out?

Reply:
Ah, I need to remove this line. As these settings were specific to the older version, I was testing things by commenting out code. Will remove this once the review is complete.

Reply:
Please do

Reply:
Done!!